commands/auth.ts

```diff
@@ -64,35 +57,52 @@ const ALLOWED_AUTH_METHODS = [
 const SUPPORTED_AUTHENTICATION_PROTOCOLS_TEXT =
   commaSeparatedValues(ALLOWED_AUTH_METHODS);
 
-exports.command = 'auth';
-exports.describe = i18n(`${i18nKey}.describe`, {
+export const command = 'auth';
+export const describe = i18n(`${i18nKey}.describe`, {
   configName: DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME,
 });
 
-exports.handler = async options => {
+type AuthArgs = CommonArgs &
+  ConfigArgs &
+  TestingArgs & {
+    authType?: string;
+    account?: string;
+  };
```

Comment:
Can use `AccountArgs` here

Reply:
Done in [be71b49](https://github.com/HubSpot/hubspot-cli/pull/1401/commits/be71b495f9f24049001fda5d7c3328c7479fcdc8)